src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -525,6 +527,39 @@ validate_args(#mrst{} = State, Args0) ->
     end.
 
 
+apply_limit(ViewPartitioned, Args) ->
+    MaxLimitType = case ViewPartitioned of
+        true -> "partition_";
```

Comment:
I think it'd be easier to understand this bit if we just type out the config names rather than doing the concatenation down below.